lang/java/gradle-plugin/README.md

```diff
@@ -0,0 +1,471 @@
+# Seeking New Maintainer
+
+For more details, see [the discussion](https://github.com/davidmc24/gradle-avro-plugin/discussions/208).
+
+If no new maintainer is found, this project will be archived on October 22, 2023 (the 10 year anniversary of this project's first public commit).
+
+# Overview
+
+This is a [Gradle](http://www.gradle.org/) plugin to allow easily performing Java code generation for [Apache Avro](http://avro.apache.org/).  It supports JSON schema declaration files, JSON protocol declaration files, and Avro IDL files.
+
+[![Build Status](https://github.com/davidmc24/gradle-avro-plugin/workflows/CI%20Build/badge.svg)](https://github.com/davidmc24/gradle-avro-plugin/actions)
+
+# Compatibility
+
+**NOTE**: Pre-1.0 versions used a different publishing process/namespace.  It is strongly recommended to upgrade to a newer version.  Further details can be found in the [change log](CHANGES.md).
+
```

Comment:
I would argue this is no longer useful, given that the pre-1.0 version was from 2017 and if people are using the Apache version, this would be doubly irrelevant.

```suggestion
```